view/adminhtml/layout/sales_order_creditmemo_new.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
```

Comment:
Rather than having ~7 XML files with duplicate content, add the XML to an arbitrary filename like `avatax_sales_totals_tax.xml` and then within each of these files use code like this:

```
<!--
/**
 * ClassyLlama_AvaTax
 *
 * NOTICE OF LICENSE
 *
 * This source file is subject to the Open Software License (OSL 3.0)
 * that is bundled with this package in the file LICENSE.txt.
 * It is also available through the world-wide-web at this URL:
 * http://opensource.org/licenses/osl-3.0.php
 *
 * @copyright  Copyright (c) 2018 Avalara, Inc.
 * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
 */
-->
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <update handle="avatax_sales_totals_tax"/>
</page>
```